anki/template/template.py

```diff
@@ -5,7 +5,17 @@
 from anki.utils import stripHTML, stripHTMLMedia
 
 # Matches a {{c123::clozed-out text::hint}} Cloze deletion, case-insensitively.
-clozeReg = r"(?si)\{\{(c)%s::(.*?)(::(.*?))?\}\}"
+# The regex should be interpolated with a regex number and creates the following
+# named groups:
+#   - tag: The lowercase or uppercase 'c' letter opening the Cloze.
+#   - content: Clozed-out content.
+#   - hint: Cloze hint, if provided.
+clozeReg = r"(?si)\{\{(?P<tag>c)%s::(?P<content>.*?)(::(?P<hint>.*?))?\}\}"
+
+# Constants referring to group names within clozeReg.
+matchGroupTag = "tag"
+matchGroupContent = "content"
+matchGroupHint = "hint"
```

Comment:
The constants are not clearly related to clozeReg, nor clearly constants. Perhaps something like CLOZEREG_TAG or CLOZE_GROUP_TAG would be clearer?

Reply:
Sure, done. (I prefer `UPPER_SNAKE` for constants, too, but didn't know if it was OK to switch convention.)

Reply:
Existing variables shouldn't be changed, but the contrib file was recently changed to allow PEP8 in newly contributed code.